balboa-agent/src/main/scala/com/socrata/balboa/agent/Config.scala

```diff
@@ -29,6 +36,18 @@ trait Config extends JMSClientConfig {
 
   def initialDelay(defaultDelay: Long = 0): Long = Configuration.get().getLong(Keys.InitialDelay, defaultDelay)
 
+  def httpOrMQ(defaultMethod: TransportType = Mq): TransportType =
```

Comment:
So when the enum name was changed, that should cascade to all the other places it was used. Like this method name, and the config file.